charts/kube-prometheus-stack/values.yaml

```diff
@@ -1865,6 +1865,26 @@ prometheusOperator:
     allowPrivilegeEscalation: false
     readOnlyRootFilesystem: true
 
+  # Enable vertical pod autoscaler support for prometheus-operator
+  verticalPodAutoscaler:
+    enabled: false
+    # List of resources that the vertical pod autoscaler can control. Defaults to cpu and memory
+    controlledResources: []
+
+    # Define the max allowed resources for the pod
+    maxAllowed: {}
+    # cpu: 200m
+    # memory: 100Mi
+    # Define the min allowed resources for the pod
+    minAllowed: {}
+    # cpu: 200m
+    # memory: 100Mi
+
+    # updatePolicy:
```

Comment:
We can set it of to off by default for sure but since enabling VPA is a conscious choice (i.e. needs to be enabled) and `updateMode: Auto` is the default VPA CR value, should we not let the customer set it to off if they want to?

Reply:
"Off" still gives recommendations but "auto" is ok for me too.
Could we at least enable that block instead having it commented out?

Reply:
Sure thing, it's pushed :)